cmd/age/age.1

```diff
@@ -0,0 +1,162 @@
+.Dd June 28, 2020
+.Dt AGE 1
+.Sh NAME
+.Nm age
+.Nd a simple, modern and secure file encryption tool
+.Sh SYNOPSIS
+.Nm
+.Fl r Ar recipient
+.Op Fl a
+.Op Fl o Ar output
+.Op Ar input
+.Nm
+.Fl -decrypt
+.Op Fl i Ar key
+.Op Fl o Ar output
+.Op Ar input
+.Sh DESCRIPTION
+In the first synopsis form,
+.Nm
+encrypts
+.Ar input
+to the specified
+.Ar recipient .
+In the second synopsis form,
+.Nm
+decrypts
+.Ar input .
+.Pp
+The following options are available:
+.Bl -tag -width Ds -compact
+.It Fl o , Fl -output Ar output
+Write the result to the file at path
+.Ar output .
```

Comment:
Something that people might want to know: Does this *overwrite* output or error out if the file already exists?

Just food for thought.